dopamine/experiment5/experiment5_statstests.jl

```diff
@@ -0,0 +1,270 @@
+
+lm(@formula(meanspeed ~ worm*bacteria), subset(trackstats, :medium=>ByRow(==("M9")))) # just buffer
+# meanspeed ~ 1 + worm + bacteria + worm & bacteria
+
+# Coefficients:
+# ───────────────────────────────────────────────────────────────────────────────────────
+#                                Coef.  Std. Error      t  Pr(>|t|)  Lower 95%  Upper 95%
+# ───────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                 269.204      10.41    25.86    <1e-78   248.721    289.688
+# worm: CB                    -64.1178     15.0256  -4.27    <1e-04   -93.6832   -34.5525
+# worm: MT                    -45.1968     14.2228  -3.18    0.0016   -73.1826   -17.211
+# bacteria: OP50             -100.851      13.4165  -7.52    <1e-12  -127.25     -74.4519
+# worm: CB & bacteria: OP50    70.0027     18.8285   3.72    0.0002    32.9544   107.051
+# worm: MT & bacteria: OP50    68.086      18.4956   3.68    0.0003    31.6927   104.479
+# ───────────────────────────────────────────────────────────────────────────────────────
+
+lm(@formula(meanspeed ~ worm*bacteria), subset(trackstats, :medium=>ByRow(==("DA")))) # just dopamine
+# meanspeed ~ 1 + worm + bacteria + worm & bacteria
+
+# Coefficients:
+# ───────────────────────────────────────────────────────────────────────────────────────
+#                                Coef.  Std. Error      t  Pr(>|t|)  Lower 95%  Upper 95%
+# ───────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                 252.881      10.0241  25.23    <1e-75   233.156   272.607
+# worm: CB                    -32.7666     13.4317  -2.44    0.0153   -59.1975   -6.33565
+# worm: MT                    -30.643      13.8936  -2.21    0.0282   -57.9827   -3.30322
+# bacteria: OP50             -100.472      12.1398  -8.28    <1e-14  -124.36    -76.5829
+# worm: CB & bacteria: OP50    20.7443     16.8004   1.23    0.2179   -12.3155   53.8041
+# worm: MT & bacteria: OP50    12.658      17.4657   0.72    0.4692   -21.7109   47.027
+# ───────────────────────────────────────────────────────────────────────────────────────
+
+lm(@formula(meanspeed ~ medium*bacteria), subset(trackstats, :worm=>ByRow(==("N2")))) # just N2
+# meanspeed ~ 1 + medium + bacteria + medium & bacteria
+
+# Coefficients:
+# ───────────────────────────────────────────────────────────────────────────────────────────
+#                                    Coef.  Std. Error      t  Pr(>|t|)  Lower 95%  Upper 95%
+# ───────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                   269.204        9.88731  27.23    <1e-69   249.71     288.699
+# medium: DA                    -16.3228      14.3767   -1.14    0.2576   -44.6689    12.0232
+# bacteria: OP50               -100.851       12.7428   -7.91    <1e-12  -125.976    -75.7266
+# medium: DA & bacteria: OP50     0.379548    17.9484    0.02    0.9831   -35.0086    35.7677
```

Comment:
N2
dopamine vs. buffer, bacteria vs. no bacteria
p = 0.9831 --> not significant